app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx

```diff
@@ -7,17 +7,17 @@ import notesApi from "apis/notes";
 const DeleteAlert = ({
   refetch,
   onClose,
-  selectedNoteIds,
-  setSelectedNoteIds,
+  selectedNoteId,
+  setSelectedNoteId,
 }) => {
   const [deleting, setDeleting] = useState(false);
 
   const handleDelete = async () => {
     try {
       setDeleting(true);
-      await notesApi.destroy({ ids: selectedNoteIds });
+      await notesApi.destroy({ ids: selectedNoteId });
       onClose();
-      setSelectedNoteIds([]);
+      setSelectedNoteId([]);
```

Comment:
Your setter name is singular. But you are setting the value as a collection. Either update the name or rethink about the value this state should be holding.